Source/DafnyCore/Backends/Dafny/AST.dfy

```diff
@@ -323,6 +323,7 @@ module {:extern "DAST"} DAST {
     UnboundedIntRange(start: Expression, up: bool) |
     Quantifier(elemType: Type, collection: Expression, is_forall: bool, lambda: Expression)
 
+  datatype FieldMutability = ConstantField | MutableField | InternalConstantField
```

Comment:
```suggestion
  datatype FieldMutability = ConstantField | ClassMutableField | ClassConstantField
```

I didn't understand "InternalConstantField" until I looked at the test.

Reply:
ClassMutableField is ok, but the last one is really internal, so it should be ClassInternalConstantField, all right?

Reply:
Thanks for adding the comment, that helps